src/TArea.cpp

```diff
@@ -701,7 +701,6 @@ void TArea::writeJsonUserData(QJsonObject& obj) const
 // Takes a userData object and parses all its elements
 void TArea::readJsonUserData(const QJsonObject& obj)
 {
-    QMap<QString, QString> results;
```

Comment:
Yeah, originally I had a generic `static` version that I'd shared from the `TMap` to the `TArea` and `TRoom` classes but then revised to access the particular classes own user data member.